libs/application/templates/no-debt-certificate/src/fields/ConfirmationField/index.tsx

```diff
@@ -0,0 +1,224 @@
+  application,
+}) => {
+  const { externalData } = application
+  const { formatMessage } = useLocale()
+  const [viewNoDebtCertificate, setViewNoDebtCertificate] = useState(false)
+  const {
+    document,
+  } = externalData.noDebtCertificate.data.debtLessCertificateResult.certificate
+
+  function renderFooter() {
+    return (
+      <>
+        <Divider />
+        <Box
+          display="flex"
+          justifyContent="flexEnd"
+          paddingTop={4}
+          marginBottom={4}
+        >
+          <Button
+            icon="arrowForward"
+            iconType="outline"
+            onClick={() => {
+              window.open(`${window.location.origin}/minarsidur`, '_blank')
+            }}
+          >
+            {formatText(m.openMySites, application, formatMessage)}
+          </Button>
+        </Box>
+      </>
+    )
+  }
+  if (viewNoDebtCertificate) {
+    return (
+      <>
+        <Box
+          display="flex"
+          marginBottom={2}
+          justifyContent="spaceBetween"
+          alignItems="center"
+        >
+          <Button
+            circle
+            icon="arrowBack"
+            onClick={() => setViewNoDebtCertificate(false)}
+            colorScheme="light"
+            title="Go back"
+          />
+          <a
+            href={`data:application/pdf;base64,${document}`}
+            download="skuldleysisvottord.pdf"
+            className={styles.linkWithoutDecorations}
+          >
+            <Button icon="download" iconType="outline" variant="text">
+              {formatText(
+                m.downloadNoDebtCertificate,
+                application,
+                formatMessage,
+              )}
+            </Button>
+          </a>
+        </Box>
+        <PdfViewer
+          renderMode="svg"
+          file={`data:application/pdf;base64,${document}`}
+        />
+        {renderFooter()}
+      </>
+    )
+  }
+
+  return (
+    <>
+      <Text variant="h2" marginBottom={4}>
+        {formatText(m.confirmation, application, formatMessage)}
+      </Text>
+      <Box marginBottom={3} paddingTop={0}>
+        <AlertMessage
+          type="success"
+          title={formatText(m.successTitle, application, formatMessage)}
+          message={
+            <Box component="span" display="block">
+              <Text variant="small">
+                {formatText(m.successDescription, application, formatMessage)}
+              </Text>
+            </Box>
+          }
+        />
+      </Box>
+      <Box
+        marginBottom={3}
+        marginTop={0}
+        background="blue100"
+        padding={4}
+        display="flex"
+      >
+        <LinkContext.Provider
+          value={{
+            linkRenderer: (href, children) => (
+              <a
+                style={{
+                  color: '#0061ff',
+                  textDecoration: 'none',
+                  boxShadow: 'inset 0 -1px 0 0 currentColor',
+                  paddingBottom: 4,
+                }}
```

Comment:
Any reason of inline style instead of putting this in the css file?

Reply:
You didn't answer the question and I don't see any code changes?

Reply:
I tend to agree with Sævar, if we cant fix this before the release I would urge you to fix it after the release just for consistencies sake.

Reply:
We should resolve the issue and move this into the css file. Feel free to ping me on slack if you want second set of eyes on the problem.

Reply:
I moved the styles to css.ts file